src/view/components/typography/body.jsx

```diff
@@ -25,7 +25,14 @@ import getDimensionStyle from "../../utils/getDimensionStyle";
  * for more info. Once React-Spectrum provides a Body component
  * that handles typography, we can get rid of this.
  */
-const Body = ({ children, size = "S", isSerif, marginTop, marginBottom }) => {
+const Body = ({
+  "data-test-id": dataTestId,
```

Comment:
These changes for adding support for `data-test-id` within `Body` and `Heading` are also in https://github.com/adobe/reactor-extension-alloy/pull/178.